air-quality-ui/src/components/single-city/SingleCity.tsx

```diff
@@ -169,6 +165,12 @@ export const SingleCity = () => {
     )
   }, [])
 
+  const selectSite = useCallback((siteName: string) => {
```

Comment:
This isn't covered as part of the unit tests but seeing as the deselect hasn't been covered previously I assume it's not practical?